src/atomizer.js

```diff
@@ -215,6 +215,10 @@ Atomizer.prototype.parseConfig = function (config/*:AtomizerConfig*/, options/*:
             // parse them and return a valid value
             values = values.split(',').map(function (value, index) {
                 var matchVal = Grammar.matchValue(value);
+                if (matchVal == null) {
+                    console.warn('Error: Invalid value "' + value + '".');
```

Comment:
It'd be helpful if the error message also included the selector so the user can more easily identify the problem